README.md

```diff
@@ -66,23 +81,65 @@ app.get('/products',
   });
 ```
 
+### API Documentation:
+
+`auth()` accepts an asynchronous function receiving a token and returning a set of claims.
+
+`requiredScopes()` accepts either an string or an array of strings.
+
+`strategies.openid` accepts the following parameters:
+
+
+| Name                | Default                            | Description                                                          |
+|---------------------|------------------------------------|----------------------------------------------------------------------|
+| issuerBaseURL       | `env.ISSUER_BASE_URL`              | URL for the token issuer.                                            |
+| allowedAudiences    | `env.ALLOWED_AUDIENCES.split(',')` | Allowed audiences for the token.                                     |
+| clockTolerance      | `5`                                | Clock tolerance in seconds for token verification, aka leeway.       |
```

Comment:
Any particular reason to use 5 secs over other value? e.g. auth0-spa-js default leeway is 60.

Reply:
That's just what it came with. Would need to update it in the code as well. We'll make that part of the EA criteria.